front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -53,8 +53,8 @@ const TimesStopsOutput = ({
       tableType={TableType.Output}
       cellClassName={({ rowData: rowData_ }) => {
         const rowData = rowData_ as PathWaypointRow;
-        const arrivalScheduleNotRespected = rowData.arrival
-          ? rowData.calculatedArrival !== rowData.arrival
+        const arrivalScheduleNotRespected = rowData?.arrival?.time
```

Comment:
If you cast on the upper line, you shouldn't need this ?

```suggestion
        const arrivalScheduleNotRespected = rowData.arrival?.time
```

Reply:
It’s not really to comply with the type, it’s a runtime safety. I think I head some crashes, but it doesn’t seem to happen if i remove it. 
let’s remove for now.